tests/simpleKVBC/TesterReplica/internalCommandsHandler.cpp

```diff
@@ -74,6 +44,7 @@ void InternalCommandsHandler::execute(InternalCommandsHandler::ExecutionRequests
                 "The message is too small: requestSize is " << req.request.size() << ", required size is "
                                                             << sizeof(SimpleRequest));
       req.outExecutionStatus = -1;
+      continue;
```

Comment:
why is this necessary actually?

Reply:
If it won't be there so it will go to the next lines and get to one of the executes. we just want to go to the next request.